docs/dataservice-read-catalog-example.md

```diff
@@ -69,55 +83,74 @@ cd examples/dataservice-read/android
 
 Alternatively, you can use the Android Studio IDE by opening the `build/examples/dataservice-read/android/build.gradle` script.
 
-After installing and running the `dataservice_read_example` apk, you should see the `Reading the partition data from the specified catalog completed successfully` message in the main UI screen if you provided correct keyId and secretKey. If you encountered an error, please check the device's logcat for the error message.
 
-### Additional notes
-
-Note, that you can run `CMake` command directly from `<olp-edge-sdk-root>/examples/dataservice-read/` folder if you have already built and installed HERE OLP Edge SDK C++ libraries for Android. Make sure that you pass the correct path to `LevelDB` library and provided the correct `EDGE_SDK_NETWORK_PROTOCOL_JAR` parameter in the `CMake` command invoked by the `build/examples/dataservice-read/android/app/build.gradle` script.
+> **Note:** you can run `CMake` command directly from the `<olp-edge-sdk-root>/examples/dataservice-read/` folder if you have already built and installed HERE OLP Edge
+SDK C++ libraries for Android. Make sure you provide the correct path to the `LevelDB`
+library and `EDGE_SDK_NETWORK_PROTOCOL_JAR` parameter in the `CMake` command,
+invoked by the `build/examples/dataservice-read/android/app/build.gradle` script.
 
 ## Building and running on iOS
 
-This example shows how to integrate the HERE OLP Edge SDK C++ libraries into a basic iOS application written in Objective-C language and shows how to read data from a catalog layer and partition.
+This example shows how to integrate the HERE OLP Edge SDK C++ libraries into a
+basic iOS application, written in Objective-C language and read data from a catalog
+layer and partition.
 
 ### Prerequisites
 
-* Setup the iOS development environment - install the `XCode` and command line tools.
-* Install external dependencies - refer to the `README.md` file located under `<olp-edge-sdk-root>/README.md`.
-* Provide correct keyId and secret key in the `example.cpp` (refer to the beginning of this README file).
+* Set up the iOS development environment by installing the `XCode` and command-line tools.
+* Install external dependencies by referring to the `README.md` file located under
+the `<olp-edge-sdk-root>/README.md`.
+* Replace the placeholders in `examples/dataservice-read/example.cpp` with your
+app access key id and secret access key.
 
 ### Build HERE OLP Edge SDK C++
 
-First, before building, you need to configure the HERE OLP Edge SDK C++ with `EDGE_SDK_BUILD_EXAMPLES` set to `ON`, (optionally) disable tests with `EDGE_SDK_ENABLE_TESTING` set to `OFF`, and specify the path to the iOS toolchain file shipped together with the SDK and located under `<olp-edge-sdk-root>/cmake/toolchains/iOS.cmake`:
+Before you build the example, complete the following:
+* Configure the HERE OLP Edge SDK C++ with `EDGE_SDK_BUILD_EXAMPLES` set to `ON`.
+* Disable tests with `EDGE_SDK_ENABLE_TESTING` set to `OFF`.
```

Comment:
This is an optional step as pointed out in the original text

Reply:
Got it